eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriterTest.java

```diff
@@ -243,6 +243,10 @@ private void checkSequenceOntology(String sequenceOntology, String reference, St
         assertEquals(sequenceOntology, infoMap.get(VARIANT_CLASS_KEY).get(0));
     }
 
+    private String surroundedByTabOrSemicolonInDataLine(String regex) {
```

Comment:
Sorry if I'm super-picky, but what about `createRegexSurroundedByTabOrSemicolonInDataLine`? The current name could sound like it checks if the regex is surrounded or not, but then it returns a string which makes it confusing.